x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/TimeSeriesRestDriver.java

```diff
@@ -363,16 +365,34 @@ private static void ensureGreen(String index) throws IOException {
     public static Integer getNumberOfSegments(RestClient client, String index) throws IOException {
```

Comment:
minor nit: what do you think about updating this to `getNumberOfPrimarySegments`? I see we only use it in 5 places, so it wouldn't be a huge change?

Reply:
Yeah I can do that. Specifically it's `getNumberOfPrimarySegmentsForTheFIrstShardWeFind` but that's getting a little wordy. 😄